sharding-proxy/sharding-proxy-backend/src/main/java/org/apache/shardingsphere/shardingproxy/backend/communication/jdbc/execute/callback/ProxySQLExecuteCallback.java

```diff
@@ -61,14 +62,17 @@
     private final boolean fetchMetaData;
     
     private boolean hasMetaData;
+
+    private SQLStatementContext sqlStatementContext;
     
     public ProxySQLExecuteCallback(final BackendConnection backendConnection, final JDBCExecutorWrapper jdbcExecutorWrapper, 
-                                   final boolean isExceptionThrown, final boolean isReturnGeneratedKeys, final boolean fetchMetaData) {
+                                   final boolean isExceptionThrown, final boolean isReturnGeneratedKeys, final boolean fetchMetaData, final SQLStatementContext sqlStatementContext) {
```

Comment:
Considering that the type of isExceptionThrown, isReturnGeneratedKeys and fetchMetaData is boolean, do you think it is better if we put sqlStatementContext in front of those paramters?